respy/tests/test_conditional_draws.py

```diff
@@ -69,7 +69,7 @@ def test_calculate_conditional_draws():
     updated_chols[1] = np.array([[1, 0, 0], [2, 3, 0], [4, 5, 6]])
 
     calculated = calculate_conditional_draws(
```

Comment:
Why do we pass in `MAX_LOG_FLOAT`into the function as an extra argument and not simply import it to the function directly?

Reply:
I still find it weird, too. It seems like guvectorize does not recognize global variables, so we had to input the variable.